feature-group-definitions/locks.yml

```diff
@@ -0,0 +1,13 @@
+name: Locks
```

Comment:
Should we call it just locks, web locks, or something else?

That API is a bit like a mutex, allowing different bits of code to avoid using the same resource at the same time, but that's something you do with the API, the API itself doesn't "lock resources" or something like that.

Reply:
Oh good the spec is "web locks." I like how this, said aloud, sounds like "web logs." Let's leave it as "locks" for now. If it's confusing, we can revisit it with developer feedback.